sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -37,6 +37,9 @@ export class CreateInstitutionAPIInDTO {
   website: string;
   @IsNotEmpty()
   regulatingBody: string;
+  @ValidateIf((e) => e.regulatingBody === "other")
+  @IsNotEmpty()
+  otherRegulatingBody: string;
```

Comment:
Not related to your PR. @andrewsignori-aot @andrepestana-aot @ann-aot we may have to eventually apply more specific validation (like @MaxLength) to the string to avoid API being by a huge string data.

Reply:
This is related to the PR @dheepak-aot.
We do not have a need to leave it for later anymore. For every new filed we need to be as specific as possible.
For this one, we can limit to the same length available for DB.

Reply:
Sounds Good. Thanks Andrew, I will add it.